pkg/test/helpers.go

```diff
@@ -92,3 +94,7 @@ func Contains(t *testing.T, container interface{}, obj interface{}) {
 	}
 	t.Errorf("no matches found in: %s", strings.Join(diffs, "\n"))
 }
+
+func FakeClock() clockwork.FakeClock {
+	return clockwork.NewFakeClockAt(time.Date(1984, time.April, 4, 0, 0, 0, 0, time.UTC))
```

Comment:
you may updated the previous default value here.

https://github.com/jonboulle/clockwork/pull/55/files#diff-4cc12df1de8f3164eac723ca9251237312941079fd075dca3907d1386a44abf1L49
```
NewFakeClockAt(time.Date(2006, time.January, 2, 15, 4, 5, 0, time.UTC))
```

Reply:
well, the previous value that we were using was
https://github.com/jonboulle/clockwork/blob/6a247c35607b17156e639c6e0606887fa6fc304f/clockwork.go#L42

```
NewFakeClockAt(time.Date(1984, time.April, 4, 0, 0, 0, 0, time.UTC))
```

Reply:
yes @vinamra28 seems correct otherwise all the tests would ave been failing

Reply:
Thanks @vinamra28 , I took the wrong reference